productivity-suite/app/legacy-app/src/worker/script.ts

```diff
@@ -138,16 +138,12 @@ gateway.registerFragment({
       params.append("listName", listName);
     }
 
-    return new Request(`${thisConfig.getBaseUrl(env)}?${params}`, request);
+    return new Request(`${workerBaseUrl}?${params}`, request);
```

Comment:
Or actually even simpler, instead of the code from line 136 onwards, you can use the approach we took in cloud-gallery:

```ts
const url = new URL(request.url);
if (listName) {
  url.searchParams.append("listName", listName);
}
return new Request(url, request));
```

Reply:
ah yeah, you're totally right, there is no need necessarily override the request base itself 🙂

Reply:
@petebacondarwin it could be all good now, please have a look 🙂 🙏